### PR TITLE
Chore: Update contributing guide

### PR DESCRIPTION
**Pull request by madcampos**
## Summary

Added more context to the contribution guide, describing how to setup things and adding things like coding guidelines and explanation of the tools used.

Also added a mini roadmap (to be discussed with @kbventures soon).

Lastly, added a "notes" file for dumping information

## Tests

N/A

## Additional information

Those changes are the result of discussions over on slack to improve documentation for people who are contributing to the project and also for adding notes on the things we discussed.

### CODE REVIEW

**Review by lylaminju — COMMENTED** [review]
Just curious, is there a reason you chose to install `dprint` manually instead of using npm? I thought using npm would help ensure consistent versions for all developers.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,26 +1,202 @@
...
+## Committing
+
+We don't enforce a committing standard, but below are some guidelines for commit messages:
+
+- Be descriptive, a commit message tells what happened to that code.
+- Commit messages should aim to "fit in a Tweet". That adds conciseness.
+- The first line of the commit is the summary, use it!
+- If more context is needed for the changes, write a summary, leave a blank line, then add a longer message as needed.
+- Commit often so you don't have a lot to commit about.
+- The changed files are part of the commit, so no need to be redundant in the message.
```
Comment (pmjuu) [inline CONTRIBUTING.md:196]: What about adding a guideline to include a prefix in commit messages, following [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)?

Comment (madcampos) [reply CONTRIBUTING.md:196]: Sure, I'll add that

**Comment by madcampos** [comment]
@pmjuu:
> Just curious, is there a reason you chose to install `dprint` manually instead of using npm? I thought using npm would help ensure consistent versions for all developers.

No particular reason, we can do that. My concern was that the `dprint` package on npm was something unrelated, but looking at it, seems like it is the right tool. I'm adding it as a dependency

**Review by lylaminju — APPROVED** [review]

**Pull request merged**